src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -2303,3 +2303,19 @@ class DropTriggerStatementSegment(BaseSegment):
         Ref("TableReferenceSegment"),
         OneOf("CASCADE", "RESTRICT", optional=True),
     )
+
+
+@postgres_dialect.segment(replace=True)
+class InsertStatementSegment(BaseSegment):
+    """An `INSERT` statement."""
```

Comment:
Can you add a reference to the Postgres 14 docs in your docstring?

Reply:
Added in the doc reference :)